frontend/src/environments/environment.on-premise.ts

```diff
@@ -0,0 +1,5 @@
+export const environment = {
+  production: true,
+  backendUrl: '192.168.1.110:5000',
```

Comment:
Try to avoid this harcoded variable

Reply:
Check this: https://www.jvandemo.com/how-to-use-environment-variables-to-configure-your-angular-application-without-a-rebuild/

Reply:
This should be really well explained in the install documentation